transport_nantes/topicblog/models.py

```diff
@@ -776,9 +776,21 @@ class TopicBlogEmailSendRecord(models.Model):
 
     """Represent the fact that we sent an email.
     """
+
+    class StatusChoices(models.TextChoices):
+        # This is the default state
+        PENDING = 'PENDING', "Pending"
```

Comment:
It's not so much the default state as the initial state.

Reply:
changed to
# This is the initial state